tests/tracer/test_context.py

```diff
@@ -86,8 +86,8 @@ def validate_traceparent(context, sampled_expected):
     "context",
     [
         Context(),
-        Context(trace_id=123, span_id=321),
-        Context(trace_id=123, span_id=321, dd_origin="synthetics", sampling_priority=2),
+        Context(trace_id=123, span_id=321, is_remote=True),
```

Comment:
Isn't `is_remote=True` the default? Or is this just for clarity

Reply:
Yup, it's true by default. I just wanted to test all the different ways the Context object could be initialized.